static/css/tips_style.css

```diff
@@ -0,0 +1,344 @@
+    border-radius: 3px;
+    width: 100%;
+}
+
+.post .author-info ul li .info {
+    float: left;
+}
+
+.post .author-info.author-info-2 ul li:first-child {
+    border-left: none;
+    padding-left: 0px;
+}
+
+.post .author-info.author-info-2 ul li .icon-box {
+    font-size: 28px;
+}
+
+.post .post-category {
+    float: left;
+    width: 100%;
+    text-align: left;
+    margin-bottom: 20px;
+}
+
+.post .post-category a {
+    margin: 0px;
+    font-size: 18px;
+    font-weight: 300;
+    color: #3b4952;
+}
+
+.post .post-category span {
+    width: 12px;
+    height: 12px;
+    display: inline-block;
+    background: #3b4952;
+    vertical-align: middle;
+    margin-right: 10px;
+}
+
+.post .post-category a:hover span {
+    background: #e74c3c;
+    color: #e74c3c;
+}
+
+.post .post-category a:hover {
+    color: #e74c3c;
+}
+
+.post .tags {
+    float: left;
+    width: 100%;
+    margin-bottom: 20px;
+}
+
+.post .tags li {
+    margin-bottom: 8px;
+    padding: 0px 2px;
+}
+
+.post .tags li a {
+    background: #ebf1f4;
+    font-size: 14px;
+    font-weight: 300;
+    border-radius: 3px;
+    padding: 4px 8px;
+    color: #3b4952;
+}
+
+.post .tags li a:hover {
+    background: #3b4952;
+    color: #fff;
+}
+
+.img-grid {
+    float: left;
+    margin-bottom: 40px;
+}
+
+.img-grid li {
+    margin: 0px;
+    float: left;
+}
+
+.post .caption h5 {
+    text-decoration: underline;
+    margin: 0px 0px 20px 0px;
+    color: #3b4952;
+    font-weight: 300;
+    font-size: 24px;
+    line-height: 30px;
+}
+
+.post .list-unstyled {
+    margin-bottom: 40px;
+}
+
+.post .list-unstyled li {
+    font-size: 16px;
+    line-height: 28px;
+    font-weight: 500;
+    color: #49545b;
+}
+
+.post .list-unstyled li i {
+    color: #a0b9ca;
+    margin-right: 15px;
+}
+
+blockquote {
+    background: #f2f6f8;
+    border: 1px solid #e5e5e5;
+    line-height: 28px;
+    margin-bottom: 40px;
+    font-size: 16px;
+    font-weight: 500;
+    color: #49545b;
+}
+
+.line-block {
+    padding: 20px 45px;
+    border-top: 1px solid #eef3f6;
+    border-bottom: 1px solid #eef3f6;
+    float: left;
+    width: 100%;
+}
+
+.post .line-block .tags {
+    margin-bottom: 0px;
+}
+
+.vt-post.post .author-info ul li {
+    padding-left: 15px;
+    padding-right: 15px;
+    float: none;
+}
+
+.vt-post.post .author-info {
+    border-radius: 0px 0px 0px 3px;
+    border-bottom: none;
+    border-right: 1px solid #ddd;
+    padding: 15px 12px 15px 12px;
+}
+
+.vt-post.post .post-img a img {
+    border-radius: 3px 0px 0px 0px;
+}
+
+.vt-post.post .caption {
+    padding: 25px 0px;
+}
+
+.post .caption h3 {
+    margin: 0px 0px 20px 0px;
+    color: #36a0e7;
+    font-weight: 300;
+    font-size: 34px;
+    line-height: 42px;
+}
+
+.md-heading {
+    font-size: 24px !important;
+    line-height: 36px !important;
+    margin-bottom: 15px !important;
+}
+
+a.nav-link {
+    display: flex;
+    color: rgba(255, 255, 255, .55);
+}
+
+#tipDate {}
```

Comment:
missing line

Reply:
In css files the formating is without a new line I think